gfx.cpp

```diff
@@ -779,6 +790,10 @@ class GfxInternal
         Parameter *parameters_ = nullptr;
         uint32_t parameter_count_ = 0;
         uint32_t vertex_stride_ = 0;
+
+        // These two params are used for RT pipelines. If one of them is set, the other one must be set as well
+        uint32_t max_payload_size_ = 0;
+        uint32_t max_attribute_size_ = 0;
```

Comment:
It will be removed, since it can be configured from shaders

Reply:
Can we remove these yet?

Reply:
Done